Rakefile

```diff
@@ -4,9 +4,14 @@ require 'html-proofer'
 
 task :test do
   sh 'bundle exec jekyll build'
-  HTMLProofer.check_directory('./_site/',
-                              check_html: true,
-                              hydra: { max_concurrency: 10 },
-                              url_ignore: [%r{https://developer.github.com}, %r{https://docs.github.com}, %r{https://help.github.com}],
-                              http_status_ignore: [429]).run
+  proofer = HTMLProofer.check_directory(
+    './_site/',
+    check_html: true,
+    hydra: { max_concurrency: 10 },
+    url_ignore: [%r{https://developer.github.com}, %r{https://docs.github.com}, %r{https://help.github.com}],
+    http_status_ignore: [429])
+  proofer.before_request do |request|
```

Comment:
might want to check if the value is set, so the local script works if they haven't set this up. Maybe the empty header isn't an issue though